src/dev/LCD.cpp

```diff
@@ -3,12 +3,30 @@
 #include <cstring>
 
 namespace EVT::core::DEV {
-LCD::LCD(EVT::core::IO::GPIO& regSelect, EVT::core::IO::GPIO& reset, EVT::core::IO::SPI& spi)
-    : regSelect(regSelect), reset(reset), spi(spi) {
+LCD::LCD(IO::GPIO& regSelect, IO::GPIO& reset, IO::SPI& spi) : regSelect(regSelect), reset(reset), spi(spi) {
     this->regSelect.writePin(EVT::core::IO::GPIO::State::LOW);
     this->reset.writePin(EVT::core::IO::GPIO::State::LOW);
 }
 
+LCD::LCD(IO::GPIO& regSelect, IO::GPIO& reset, IO::SPI& spi, uint8_t numberOfSections, uint8_t sectionsPerRow,
+         uint8_t sectionHeight) : regSelect(regSelect), reset(reset), spi(spi) {
+    this->regSelect.writePin(EVT::core::IO::GPIO::State::LOW);
+    this->reset.writePin(EVT::core::IO::GPIO::State::LOW);
+
+    if (sectionsPerRow > MAX_SECTION_PER_ROW)
```

Comment:
We normally always use brackets with our if statements, even if it's only 1 line